MetaMask/metamask-extension
# Adding Bahamut logo for the MetaMask UI

### **Explanation**

Bahamut is a layer 1, EVM-based blockchain solution with a novelty consensus PoSA (Proof of Stake and Activity). Activity is a new parameter that is represented by the gas spent when interacting with a validator-deployed smart contract. This activity is later taken into account and increases the chance to become a block proposer in future epochs. The native token of Bahamut is FTN. 

Bahamut has over 100,000 users that use FTN for transfers, gaming, and smart contract interactions. 
Currently, users are fully able to add Bahamut to MetaMask and use all of the features. However, we would love to have the proper FTN logo showing up in the corresponding places in the UI. Please help us with this. 


## **Manual testing steps**

1. Add Bahamut network to MetaMask
2. Switch to Bahamut network
3. Check if the logo shows up on network display as well as token display
4. Try to transfer FTN token and verify that FTN logo is displayed.
5. Open the Blockchain network switcher dropdown in the upper left corner, and verify that FTN logo is displayed.


## **Screenshots**

This PR is mostly UI-related, to add and display the Bahamut FTN logo in proper fields.

### **Before**
<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/2fa135b9-6d5a-4245-b28a-105454e8ae8f" style="max-width: 100%;">
<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/e70c94b2-271c-4b6e-bb83-25a3b9da7148" style="max-width: 100%;">
<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/3c1b7a34-0d99-4899-b690-4f69ed255754" style="max-width: 100%;">

### **After**

<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/3634c7a6-ceb7-4471-9c38-9f8fe7a2e72d" style="max-width: 100%;">
<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/9970399e-1dec-4cae-8375-de0e6fb30375" style="max-width: 100%;">
<img width="400" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/127204391/c9e4e375-b7a5-4b58-908e-b9e16e03ad30" style="max-width: 100%;">

## **Pre-merge author checklist**



## **Pre-merge reviewer checklist**
